src/api_proxy/service_control/request_builder.cc

```diff
@@ -1276,6 +1292,12 @@ Status RequestBuilder::AppendByConsumerOperations(
     Timestamp current_time) const {
   Operation* op = request->add_operations();
   SetOperationCommonFields(info, current_time, op);
+  if (info.api_consumer_identity == identity::ApiConsumerIdentity::VERIFIED) {
```

Comment:
We need to add "set_consumer_id" for FillAllocateQuotaRequest

Reply:
We do that here: https://github.com/GoogleCloudPlatform/esp-v2/pull/295/files/111cf2d08002a6d881b628c951c60bf592de2591#diff-cbee4bff4243945b26c186374c3889fdR1141

We allow any API key, doesn't have to be verified valid.

Reply:
I see. I missed that. Thanks